ocp_resources/data_import_cron.py

```diff
@@ -59,8 +59,8 @@ def to_dict(self) -> None:
             if self.image_stream and self.url:
                 raise ValueError("imageStream and url cannot coexist")
 
-            if not self.pull_method:
```

Comment:
The file needs to be regenerated https://github.com/RedHatQE/openshift-python-wrapper/blob/main/class_generator/README.md